pkgs/development/libraries/gnutls/generic.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation {
 
   preConfigure = "patchShebangs .";
   configureFlags =
-    lib.optional stdenv.isLinux "--with-default-trust-store-file=/etc/ssl/certs/ca-certificates.crt"
+    lib.optional stdenv.isLinux "--with-default-trust-store-file=${cacert}/etc/ssl/certs/ca-bundle.crt"
```

Comment:
Does this still allows to override the certificate store?

Reply:
No, it does not allow to override the cert store. As far as I understand, the only way to make this happen is to patch in `SSL_CERT_FILE` support into GnuTLS.

Reply:
I don't feel comfortable with this solution. Yes, it's guaranteed to work on any system. But it kills any flexibility to add custom certs and makes GnuTLS unusable in environments that require them.

Unless we want to patch GnuTLS, I would suggest to leave it as is and document that GnuTLS  expects the certs in `/etc/ssl/certs/ca-certificates-crt`, so on non-NixOS systems the user should copy or symlink the cert bundle there. Of course that requires root access...

Reply:
Agree, I will patch GnuTLS then.